var/spack/repos/builtin/packages/py-dictdiffer/package.py

```diff
@@ -12,6 +12,7 @@ class PyDictdiffer(PythonPackage):
     homepage = "https://github.com/inveniosoftware/dictdiffer"
     pypi = "dictdiffer/dictdiffer-0.8.1.tar.gz"
 
+    version("0.9.0", sha256="17bacf5fbfe613ccf1b6d512bd766e6b21fb798822a133aa86098b8ac9997578")
```

Comment:
Just for our peace of mind, these modifications in `builtin` packages come from upstream? (@heerener for awareness)

Reply:
Should I wait for https://github.com/spack/spack/pull/40933 to be merged or we can already merge this one?

Reply:
We can probably merge now, then file a new PR to backport the upstream modifications later.

I have some time, so if you want I can file upstream PRs for the version bumps.

Reply:
Great, thanks!

Reply:
@heerener https://github.com/spack/spack/pull/40933 has been merged in upstream, is there anything else I should do?

Reply:
As there were no further modifications I think you're good!

Reply:
Perfect, thanks!